packages/trader/src/Modules/Trading/Components/Form/TradeTypeInfo/trade-type-info-item.jsx

```diff
@@ -60,7 +60,7 @@ const TradeTypeInfoItem = ({
                                     id={`dt_contract_info_${item.value}_button`}
                                     className='trade-type-info-dialog__choose-button'
                                     onClick={() => onSubmitButtonClick(type)}
-                                    text={localize('Choose')}
+                                    text={`${localize('Choose')} ${item.text}`}
```

Comment:
@mustofa-binary should we use a placeholder inside the localize string instead? like `text={localize('Choose {{trade_type}}', { trade_type: item.text })}`

Reply:
i thought, that placeholder functionality is for `Localize` function, not `localize`. let me check